### ISSUE TITLE
Some questions from beginners

### ISSUE THREAD

**Issue by Betristor** [Calysto/matlab_kernel#162]
Hi, I just finnished the installation of matlab kernel for jupyter and jupyterlab. But I do have some questions which are not solved from documentations.

1. What's the difference between kernel ```matlab``` and ```matlab(connection)```?
![image](https://user-images.githubusercontent.com/55777837/169978750-31af12b4-d47a-4043-ab9d-bc958b3f267b.png)

2. How could I use those toolboxs downloaded from the internet? Like yalmip, cvx and so on.
3. Is there a language server for matlab kernel? Cause lsp has problem detecting the language.
![image](https://user-images.githubusercontent.com/55777837/169979269-f7b02b61-6fec-4874-bce7-4c4f51315dab.png)

**Comment by blink1073** [comment]
Hi @Betristor,  

1. `matlab(connection)` is used to connect to an already-running Matlab, while `matlab` starts a new Matlab process.
2. Installed toolboxes should work the same as they would if you launched Matlab
3. I don't believe there is a Language Server for matlab yet: https://microsoft.github.io/language-server-protocol/implementors/servers/

**Comment by Betristor** [comment]
Thanks for your reply!
I tried to change the kernel type between ```matlab(connection)``` and ```matlab``` and each time the kernel will initiate as I changed, which means they could not be alternatives for the other.
And I do check the list of current existing language server protocols and no sight of matlab. But I think it's a briliant idea to develop one for matlab for this kernel offers a convenient way to interact with matlab on browser without annoying GUI on ubuntu server.
